projects/gnu.org/binutils/package.yml

```diff
@@ -0,0 +1,54 @@
+distributable:
+  url: https://ftp.gnu.org/gnu/binutils/binutils-{{ version.raw }}.tar.xz
+  strip-components: 1
+
+versions:
+  #TODO HTML listing: https://ftp.gnu.org/gnu/binutils/
+  - v2.39
+
+build:
+  dependencies:
+    tea.xyz/gx/make: '*'
+    gnu.org/bison: '*'
+    gnu.org/texinfo: '*'
+    darwin:
+      tea.xyz/gx/cc: c99
+    linux:
+      gnu.org/gcc: '*'
+  script: |
+    # FIXME: bootstrap - we need `as` from binutils to build with gcc
+    if test "{{hw.platform}}" = "linux"; then
+      if command -v sudo; then
+        sudo apt-get --yes install binutils
```

Comment:
We'll formalize a “bootstrap” key at some point. Which is allowed to source stuff from other places.

Reply:
But yeah, let's not make a habit of locking ourselves to `apt` based systems.

Reply:
agreed. this is to bootstrap with our current infrastructure, and will be removed once tea can provide gcc for bootstrapping.